src/test/java/com/google/api/codegen/testdata/nodejs_main_library.baseline

```diff
@@ -102,6 +88,8 @@ var ALL_SCOPES = [
  *   Custom function to obtain the credentials.
  * @param {grpc.ClientCredentials} opts.sslCreds
  *   A ClientCredentials for use with an SSL-enabled channel.
+ * @param {Object} opts.grpc
```

Comment:
Note from offline discussion: this parameter only needs to be used to avoid type mismatches if the user wants to pass in custom gRPC objects (e.g., the channel or credentials), since there will be multiple gRPC modules (corresponding to different types) if this option is not enabled. It shouldn't need to be used in the simple case.